products/interna/open-access-ausgabe/index.qmd

```diff
@@ -27,6 +27,8 @@ other-links:
 
 Die Publikationsreihe *Stadt.Geschichte.Basel* präsentiert in zehn Bänden (Bände 1–9 sowie ein Abschlussband) eine umfassende Aufarbeitung der Basler Stadtgeschichte von den frühesten Anfängen bis zur Gegenwart. Ein zentrales Anliegen des Projekts ist es, historische Forschungsergebnisse möglichst breit zugänglich zu machen und für unterschiedliche Interessensgruppen nutzbar zu halten. Um dies zu erreichen, werden die Bände [in einer Open-Access-Version veröffentlicht](https://openaccess.stadtgeschichtebasel.ch/), die kostenlos im Internet zur Verfügung steht.
 
+[![Screenshot der Open-Access-Plattform](../../../assets/img/emono_screenshot.png)](https://emono.unibas.ch/stadtgeschichtebasel)
```

Comment:
For consistency we should use absolute paths `(/assets/img/emono_screenshot.png)](https://emono.unibas.ch/stadtgeschichtebasel)`